references/Subgraph/Endpoints.md

```diff
@@ -0,0 +1,17 @@
+sidebar_position: 1
+title: Endpoints
+
+---
+
+The Balancer Subgraph indexes data on the Balancer smart contracts with a GraphQL interface. It updates data in response to function calls and contract events to maintain data on the `Vault`, `Pools`, `AssetManagers` etc, to power front-end apps and integrations.
+
+\*\*Note that Balancer has not yet migrated to the new Subgraph Studio mainnet. Any Balancer subgraphs appearing there should not be considered "official" deployments. Use at your own risk!
+
+| Network          | Subgraph URL                                                                                                                                                         |
+| ---------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Ethereum Mainnet | [https://thegraph.com/legacy-explorer/subgraph/balancer-labs/balancer-v2](https://thegraph.com/legacy-explorer/subgraph/balancer-labs/balancer-v2)                   |
+| Polygon          | [https://thegraph.com/legacy-explorer/subgraph/balancer-labs/balancer-polygon-v2](https://thegraph.com/legacy-explorer/subgraph/balancer-labs/balancer-polygon-v2)   |
+| Arbitrum         | [https://thegraph.com/legacy-explorer/subgraph/balancer-labs/balancer-arbitrum-v2](https://thegraph.com/legacy-explorer/subgraph/balancer-labs/balancer-arbitrum-v2) |
+| Kovan            | [https://thegraph.com/legacy-explorer/subgraph/balancer-labs/balancer-kovan-v2](https://thegraph.com/legacy-explorer/subgraph/balancer-labs/balancer-kovan-v2)       |
+| Rinkeby          | [https://thegraph.com/legacy-explorer/subgraph/balancer-labs/balancer-rinkeby-v2](https://thegraph.com/legacy-explorer/subgraph/balancer-labs/balancer-rinkeby-v2)   |

```

Comment:
could you remove these links? we're deprecating Kovan and Rinkeby.

Reply:
I deleted these out